### ISSUE TITLE
component.prototype. functions not accessible inside content blocks

### ISSUE THREAD

**Issue by pahans** [meteorhacks/flow-components#32]
I have following nested components inside my code
```js
{{#render component="loading" isLoading=stateFn$isProfileLoading}}
  {{>render component="hbar"
    onSelect=action$onHarbarSelect
  }}
{{/render}}
```
when action onHarbarSelect is triggered

```js
component.prototype.getSortedValueType = function () {
   ...
};

component.action.onHarbarSelect = function(id) {
  console.log(this.getSortedValueType)
};
```

console.log about prints `undefined` for above code. without the content block its working fine.

**Comment by arunoda** [comment]
Okay. Seems like contentBlock and scope issue. It'll have a check.

**Comment by arunoda** [comment]
Fixed with 3ee61d2

**Comment by arunoda** [comment]
Fixed with 3ee61d2863bcc9519f15fa833a09165481738fd7
